javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,124 @@
+/* Javascript week1 homework*/
+
+//Start of A future age calculator
+console.log("A future age calculator");
+const yearOfBirth = 1990;
+const yearFuture = 2040;
+const age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`);
+//End of future age calculator
+
+//Start of A dog age calculator
+console.log("A dog age calculator");
+const dogYearOfBirth = 2000;
+const dogYearFuture = 2030;
+const dogYear = 50;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
+	console.log(`Your dog will be ${dogYear} in ${dogYearFuture}`);
+} else {
+	console.log(
+		`Your dog will be ${dogYearFuture - dogYearOfBirth} in ${dogYearFuture}`
+	);
+}
+// End of A dog age calculator
+
+// Start of Housey pricey (A house price estimator)
+const peterHouse = {
+	width: 8,
+	depth: 10,
+	height: 10,
+	gardensize: 100,
+	house_cost: 2500000,
+};
+
+const juliaHouse = {
+	width: 5,
+	depth: 11,
+	height: 8,
+	gardensize: 70,
+	house_cost: 1000000,
+};
+
+function calculateVolumeInMeters(width, height, depth) {
+	volumeInMeters = width * height * depth;
+	return volumeInMeters;
+}
+
+function calculateHousePrice(volumeInMeters, gardensize) {
+	housePrice = volumeInMeters * 2.5 * 1000 + gardensize * 300;
+	return housePrice;
+}
+
+let peterHouseVolume = calculateVolumeInMeters(
+	peterHouse.width,
+	peterHouse.height,
+	peterHouse.depth
+);
+let juliaHouseVolume = calculateVolumeInMeters(
+	juliaHouse.width,
+	juliaHouse.height,
+	juliaHouse.depth
+);
+
+let peterHousePrice = calculateHousePrice(
```

Comment:
You don't need so many functions to make this logic work. Instead, you can put the whole thing inside only one function, that gets some parameters and apply the logic inside it. Like that, your code will be cleaner, more readable and you avoid one-line logic functions.

Reply:
i will try as per your suggestion to put all thing inside one function. Thank you.